db/migrate/20200820174703_add_partial_target_id_index_to_reviewables.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPartialTargetIdIndexToReviewables < ActiveRecord::Migration[6.0]
+  def change
+    add_index :reviewables, [:target_id], where: "target_type = 'Post'"
```

Comment:
Just to err on the side of caution, I think it'll be better to add the index here concurrently. Another suggestion I have is to add a custom name for this index because the default name Rails generates will not indicate anything about the filter. We do annotate the index in our code but if someone is looking through the index in the DB, it'll seem like an index on the `target_id`.